pkg/threescale/proxy_conf.go

```diff
@@ -368,10 +368,18 @@ func GetFromRemote(cfg *config.Params, c *sysC.ThreeScaleClient, report reportMe
 	elapsed := time.Since(start)
 
 	go func() {
-		if apiErr, ok := err.(sysC.ApiErr); ok {
-			report(cfg.SystemUrl, metrics.NewLatencyReport("", elapsed, cfg.SystemUrl, metrics.System),
-				metrics.NewStatusReport("", apiErr.Code(), cfg.SystemUrl, metrics.System))
+		var httpResponseCode int
+		if err == nil {
+			httpResponseCode = 200
```

Comment:
this needs recovering the actual code from the client https://github.com/3scale/3scale-porta-go-client/issues/18

Reply:
Yes. Will make the update and update the dependency ASAP